docs/new.rst

```diff
@@ -15,6 +15,43 @@ It also features a new parser enabling clearer error messages, greater language
 
 See :ref:`porting to Scenic 3` for tools to help migrate existing 2D scenarios.
 
+Scenic 3.1.0
+------------
+
+Major new features:
+
+	* New MetaDrive simulator interface for running Scenic scenarios in MetaDrive (see :mod:`scenic.simulators.metadrive`).
```

Comment:
Let's link to the MetaDrive section of the "Supported Simulators" page instead, since that's a better starting point. You can change the text to "Added an interface to the MetaDrive driving simulator." and have "interface" be the link.